src/helpers/report-helpers.ts

```diff
@@ -0,0 +1,123 @@
+import axios from 'axios'
+
+// eslint-disable-next-line import/no-cycle
+import { ReportsApiClientV20210630 } from '../api-clients/reports-api-client-v20210630'
+import { AMAZON_US_MARKETPLACE_ID } from './constants'
+import { sleep, tabDelimitedToArray } from './utils'
+
+interface ReportParameters {
+  startDate: string
+  endDate: string
+  marketplaceIds?: string[]
+}
+
+interface LatestReportOptions {
```

Comment:
```suggestion
export interface LatestReportOptions {
```

export any interfaces/types that are part of method inputs/outputs so the user can use them